card/lib/generators/card/card_generator.rb

```diff
@@ -0,0 +1,33 @@
+# -*- encoding : utf-8 -*-
+
```

Comment:
This one is the new one to generate a migration that adds cards to your app. I'm not sure if this runs the way it should. Ideally, this is an add-on to rails generate (rails generate card [args]) when the card gem is installed. The rails app should need decko-rails and card, but not wagn. We'll need to address how assets will work in this case so that the developer can integrate assets from these two gems with their asset frameworks.